docs/user-guide/install/pe/upgrade-instructions.md

```diff
@@ -371,6 +371,12 @@ C:\thingsboard>upgrade.bat --fromVersion=3.2.0
 net start thingsboard
 ```
 
+### Troubleshooting 
```

Comment:
Move this to a separate template and use it. Will minimize the copy-paste.

Reply:
Done